### ISSUE TITLE
i wanted to add like a chat section 

### ISSUE THREAD

**Issue by san0808** [codeforcauseorg/edu-client#132]
In this chat section we will be able to share files ask doubts different rooms for different classes ,i was thinking f trying to add even a direct personal doubt message too
i would like more ideas if any to create and 
can i work on this @mentor

**Comment by san0808** [comment]
@KeenWarrior @Abhishek-kumar09 pls reply

**Comment by Abhishek-kumar09** [comment]
yes chat section is something that we had thought, if you want to work you can start working in it.
